process_report/tests/unit_tests.py

```diff
@@ -354,39 +506,13 @@ def test_apply_credit_0002(self):
         dataframe = process_report.apply_credits_new_pi(
             self.dataframe, self.old_pi_file
         )
+        dataframe = dataframe.astype({"Credit": "float64", "Balance": "int64"})
```

Comment:
Only using integers and floats makes this test not representative. Everything should be `Decimal`. But this can be changed in a follow-up.